### DISCUSSION TITLE
12 Button Mouse Supported on Linux?

### DISCUSSION THREAD

**Discussion by SarahSzabo** [AntiMicroX/antimicrox#819] [Q&A]
Hello everyone, I recently discovered that the 12 button mouse (RedDragon M901) I got wasn't supported on Antimicrox on Linux.

Are there any decent 12 button mice which are supported on Antimicrox on Linux?
Thanks!

**Comment by pktiuk** [comment]
As answered in https://github.com/AntiMicroX/antimicrox/issues/818

>  I think you should use for Piper for this purpose, not AntiMicroX https://github.com/libratbag/piper/  

Piper is an Utility used for mapping buttons on custom mouses
